RIOT-OS/RIOT
# doc/html: Fix scrollTo issue of doxygen

Fixes the issue described in #15503

### Contribution description

add a scrollTo for JQuerry since its selector does not forward it

### Testing procedure

Build the doc and use the Performance inspector of your browser to check if the issue described in #15503 is  happening 

see a build version on
https://kfessel.github.io/RIOT/html/


### Issues/PRs references
Fixes: #15503
